near-contract-standards/src/fungible_token/mod.rs

```diff
@@ -7,5 +7,7 @@ pub mod receiver;
 pub mod resolver;
 pub mod storage_impl;
 
+pub use crate::fungible_token::core::FungibleTokenCore;
```

Comment:
Let's add module-level documentation with a link to the example, so when developers visit [the doc](https://docs.rs/near-contract-standards/latest/near_contract_standards/fungible_token/index.html), they can quickly find the relevant information.